docs/tutorials/setting-up-ethernet-over-usb-on-linux-command-line.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Setting Up Ethernet Over USB on Linux with Command Line
+subject: Ethernet Over USB
+---
+
+1.  On the EV3, first verify that the CDC driver is enabled. In brickman,
+    open the *USB* settings and make sure *CDC* is selected and active.
+
+    {% include screenshot.html source="/images/brickman/usb-cdc-active.png" %}
+
+2.  Then, in *Networking*, make sure *Gadget* is enabled.
+
+    {% include screenshot.html source="/images/brickman/networking-technologies-gadget-selected.png" %}
+
+3.  On your host computer, with the Ev3 connected via USB, open a *Terminal* and run
+
+        user@host:~$ ifconfig | grep usb
+  
+    Assuming there is only ```usb0``` the output should look like:
```

Comment:
Code snippets actually only need one backtick like this: ``` `usb0` ``` (unless the snippet contains a backtick as well like in this comment).